### PR TITLE
Add `observableQuery` `currentResults`

### PR DESCRIPTION
**Pull request by tmeasday**
Sub PR of #633

### CODE REVIEW

File: src/ObservableQuery.ts [hunk 1]
```diff
@@ -204,4 +204,15 @@ export class ObservableQuery extends Observable<ApolloQueryResult> {
       });
     });
   }
+
+  public currentResult(): ApolloQueryResult {
+    const queryStoreValue = this.queryManager.getApolloState().queries[this.queryId];
+
+    if (!queryStoreValue || queryStoreValue.loading) {
+      return { data: {}, loading: true };
```
Comment (stubailo) [inline src/ObservableQuery.ts]: This doesn't match the returnPartialData contract - we need to actually read from the store here, which is what react-Apollo does.

Comment (tmeasday) [inline src/ObservableQuery.ts]: So you are saying that if `options.returnPartialData` is true and `queryStoreValue.loading` is true, we should return some data + loading here?

Comment (stubailo) [inline src/ObservableQuery.ts]: Yeah.

Comment (tmeasday) [inline src/ObservableQuery.ts]: This is now fixed. See the test and code changes.

Comment (tmeasday) [inline src/ObservableQuery.ts]: Although I need some further changes for `react-apollo`. Going to try and document this a bit better.

**Comment by tmeasday** [comment]
Closing in favor of https://github.com/apollostack/apollo-client/pull/709

**Pull request closed without merging**